Makefile

```diff
@@ -127,6 +127,14 @@ server-android:
 		-o bin/viam-server-$(BUILD_CHANNEL)-android-aarch64 \
 		./web/cmd/server
 
+# change this to android/arm64 if you're testing locally and want faster builds
+APK_ARCH ?= android/arm64,android/amd64
+
+droid-rdk.aar:
+	# creates an android library that can be imported by native code
+	gomobile bind -v -target $(APK_ARCH) -androidapi 28 -tags no_cgo -o $@ ./web/cmd/droid
```

Comment:
When trying this out on a PR on RDK, I try to do `make droid-rdk.aar` and I get the following error:

```
gomobile bind -v -target android/arm64,android/amd64 -androidapi 28 -tags no_cgo -o droid-rdk.aar ./web/cmd/droid
no exported names in the package "./web/cmd/droid"
```
There are no files in web/cmd/droid in this PR, is `droid.go` supposed be included in this PR?

Reply:
ack -- was going to do that as a follow up PR to make the review smaller

thanks for catch

I'll add it, + test the command before resubmitting